src/index.html

```diff
@@ -1,18 +1,62 @@
 <!doctype html>
-<html lang="en">
+<html
+  lang="en"
+  class="product-card"
+>
   <head>
     <meta charset="UTF-8" />
     <meta
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;400;500;600;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body class="product-card__page">
+    <section
+      class="product-card__all"
+      data-qa="card"
+    >
+      <div class="product-card__img"></div>
+      <article class="product-card__info">
+        <p class="product-card__name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </p>
+        <p class="product-card__code">Product code: 195434</p>
+        <article class="product-card__rating">
```

Comment:
Nested `<article>` tags are not usually recommended. Since the `product-card__rating` seems to be a part of the `product-card__info` block, it would be more semantically correct to use a `<div>` tag here instead of an `<article>`.